src/subtle.ts

```diff
@@ -1,3 +1,2 @@
-const isBrowser = typeof window !== 'undefined'
-// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-var-requires
-export const subtle = isBrowser ? window.crypto.subtle : (require('crypto').webcrypto.subtle as SubtleCrypto)
+// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports
+export const subtle = window.crypto?.subtle || (require('crypto').subtle as SubtleCrypto)
```

Comment:
`window` is actually defined in RN so this check wasn't working - better to just check if `window.crypto.subtle` exists